entities.py

```diff
@@ -13,15 +15,40 @@ def __init__(self, name, pos, type):
         self.pos = pos
         self.type = type
 
+    @abstractmethod
+    def move(self, direction): pass
+
+    @abstractmethod
+    def attack(self, other): pass
+
     def __repr__(self):
         return f"{self.name} <{self.type}>@{self.pos}"
 
 
-class Character(Entity, ABC):
+class PlayerBehavior(metaclass=Singleton):
 
-    @abstractmethod
-    def __init__(self, name, pos, type):
-        super().__init__(name, pos, type)
+    def take_turn(self, actor):
+        # FIXME Placeholder
+        print(f"Player {actor.name} is taking its turn...")
+
+
+class Actor(Entity):
+
+    """
+    An actor is an entity that can perform actions.
+    """
+
```

Comment:
We could use sphinx-style or google-style docstrings as a convention, and to be consistent with what we already have. You'd have to get rid of the first empty line, and since the docstring only contains a brief summary, it can just be a one-liner. See some [examples](https://pythonhosted.org/an_example_pypi_project/sphinx.html#full-code-example).

Reply:
I don't really like sphinx-style, it's a lot of very delicate syntax, I don't like google-style either but it's simpler. And btw, if you line break the first line of a docstring, it's represented the same as without a linebreak, as per PEP8, so with or without a linebreak it's represented as a summary.

Reply:
The cool thing about sphinx is that you can auto-generate docs, like Doxygen does for C++. But since you don't like sphinx or google styles, which style should we use?